### PR TITLE
Add renewed headers

### PR DESCRIPTION
**Pull request by yaron2**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -26,7 +26,6 @@ require (
 	gopkg.in/yaml.v2 v2.3.0
 	helm.sh/helm/v3 v3.4.0
 	k8s.io/api v0.20.0
-	k8s.io/apiextensions-apiserver v0.20.0
```
Comment (mukundansundar) [inline go.mod:29]: why is this removed?

Comment (yaron2) [reply go.mod:29]: It's the result of `go mod tidy`.

**Review by mukundansundar — APPROVED** [review]
